src/styles/classes/fontClasses.tsx

```diff
@@ -0,0 +1,7 @@
+export const fontSizeClasses: { [key: string]: string } = {
+  small: "text-sm"
+};
+
+export const fontWeightClasses: { [key: string]: string } = {
+  medium: "font-medium"
+};
```

Comment:
I've taken this object-oriented approach to avoid string concatenation while allowing dynamic classes. I'm new to Tailwind, so I am open to alternate solutions here/advice on best implementing re-usable components & atoms.